### PR TITLE
Fix bug in getElementTitle method

### LINKED ISSUE #93: getElementTitle is broken
returns element name instead of element title

### PR DESCRIPTION
**Pull request by 0I0u3uk**
Fixed #93

### CODE REVIEW

File: src/main/java/ru/sbtqa/tag/pagefactory/Page.java [hunk 1]
```diff
@@ -808,6 +808,11 @@ public String getElementTitle(WebElement element) {
         for (Map.Entry<Field, String> entry : PageFactory.getPageRepository().get(this.getClass()).entrySet()) {
             try {
                 if (Core.getElementByField(this, entry.getKey()) == element) {
+                    if (entry.getKey().getAnnotation(ElementTitle.class) != null
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: too complicated.
extract entry.getKey().getAnnotation(ElementTitle.class) to variable and use it.

File: src/main/java/ru/sbtqa/tag/pagefactory/Page.java [hunk 2]
```diff
@@ -808,6 +808,11 @@ public String getElementTitle(WebElement element) {
...
+                            && entry.getKey().getAnnotation(ElementTitle.class).value() != null
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: duplicated nullcheck.

File: src/main/java/ru/sbtqa/tag/pagefactory/Page.java [hunk 3]
```diff
@@ -808,10 +808,9 @@ public String getElementTitle(WebElement element) {
         for (Map.Entry<Field, String> entry : PageFactory.getPageRepository().get(this.getClass()).entrySet()) {
             try {
                 if (Core.getElementByField(this, entry.getKey()) == element) {
-                    if (entry.getKey().getAnnotation(ElementTitle.class) != null
-                            && entry.getKey().getAnnotation(ElementTitle.class).value() != null
-                            && "".equals(entry.getKey().getAnnotation(ElementTitle.class).value())) {
-                        return entry.getKey().getAnnotation(ElementTitle.class).value();
+                    ElementTitle elementTitle = entry.getKey().getAnnotation(ElementTitle.class);
+                    if (elementTitle != null && !elementTitle.value().equals("")) {
```
Comment (clicman) [inline src/main/java/ru/sbtqa/tag/pagefactory/Page.java]: elementTitle.value().isEmpty()

**Comment by 0I0u3uk** [comment]
@clicman was fixed

**Pull request merged**